public_html/storefront/view/default/template/pages/checkout/confirm.tpl

```diff
@@ -156,13 +156,13 @@
 						&nbsp;
 						<small title="<?php echo $option['title']?>"> - <?php echo $option['name']; ?> <?php echo $option['value']; ?></small>
 					<?php } ?>
-					<?php echo $this->getHookVar('checkout_confirm_product_'.$product['product_id'].'_additional_info_1'); ?>
+					<?php echo $this->getHookVar('checkout_confirm_product_'.$product['key'].'_additional_info_1'); ?>
 				</td>
 				<td><?php echo $product['price']; ?></td>
 				<td><?php echo $product['quantity']; ?></td>
 				<td class="checkout_heading"><?php echo $product['total']; ?></td>
 			</tr>
-		<?php echo $this->getHookVar('checkout_confirm_product_'.$product['product_id'].'_additional_info_2'); ?>
+		<?php echo $this->getHookVar('checkout_confirm_product_'.$product['key'].'_additional_info_2'); ?>
```

Comment:
did you test this hooks with fast_checkout extension? We plan to remove these tpls in the future and replace with fast_checkout

Reply:
I just checked that fast checkout already use $product['key'] in the order_summary template file.